### PR TITLE
Implementação de métodos adicionais para busca.

### PR DESCRIPTION
**Pull request by ajmiciano**
Implementação de alguns métodos adicionais para a implementação e consulta ao akeneo.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    akeneo (1.6.0)
+    akeneo (1.6.1)
```
Comment (schnika) [inline Gemfile.lock]: Thanks for your contribution. This would be `1.7.0` because it is not a hotfix/patch or bug but a minor release

Comment (ajmiciano) [reply Gemfile.lock]: Sorry, I had not seen the comment. Actually I had done the PR for the wrong repository. Sorry I have not seen you before.

**Comment by schnika** [comment]
@ajmiciano tell me when you are ready for the merge. I can see you are still enhancing the code. :)

**Comment by ajmiciano** [comment]
> @ajmiciano tell me when you are ready for the merge. I can see you are still enhancing the code. :)

Ok, tks! I Will close and reopen when finish.

**Pull request closed without merging**